.circleci/config.yml

```diff
@@ -245,8 +244,7 @@ jobs:
             # This should usually be a no-op (fully contained in the Yarn cache above), but we
             # include it to be safe since `yarn test` won't auto-install missing modules.
             yarn install --immutable
-            TESTFILES=$(circleci tests glob "src/**/*.test.ts*" | circleci tests split --split-by=timings)
-            yarn test:circleci $TESTFILES
+            yarn test:ci
```

Comment:
`yarn test:circleci` called into `yarn test:ci` with a limit on `maxWorkers`. Now that we have no parallelism, there's no need for the limit.